### ISSUE TITLE
Firefox zoom level messes up resizes

### ISSUE THREAD

**Issue by alerque** [glacambre/firenvim#864]
In [a comment on another issue](https://github.com/glacambre/firenvim/issues/860#issuecomment-756922515) I brought up something I thought might be related but turns out to be completely unrelated.

I'm on a HiDPI setup and my *default* Firefox zoom level is set to 150% (Ctrl+0 resets me to that). The initial size of Firenvim editor windows is correct and matches the textarea it is replacing, but any subsequent resize events cause it to be undersized. By all appearances the scale is off by 30% as if some calculation is just not taking the 150% page zoom into account at all and drawing at what would have been 100%.

![image](https://user-images.githubusercontent.com/173595/104087378-4d33fe80-5270-11eb-9132-9a577bcba4c6.png)

If I zoom out to 100% resize events no longer cause trouble and everything works fine.

**Comment by glacambre** [comment]
Thanks, can reproduce. I'll try to fix it today :).

**Comment by glacambre** [comment]
Fixed on master, will publish a new release tomorrow (the reason for that is that webextensions are updated only once a day in chrome/firefox and I already published a release today). Thanks for reporting this bug!

**Comment by glacambre** [comment]
I submitted v0.2.2 to the firefox and chrome add-on stores, let me know if you still experience issues :)

**Comment by alerque** [comment]
Hey that looks better! Thanks.
